### ISSUE TITLE
This should be encodeURIComponent

### ISSUE THREAD

**Issue by facingmonday** [cosmicjs/cosmic-sdk-js#25]
An ampersand in the query string was breaking the request.

For example
```
cosmic.objects.find({
  type: 'type',
  title: 'Something & SomethingElse'
})
    .props([
      'id',
      'title',
    ])
    .limit(1);
```

Will return: 'Query must be valid JSON string.'

encodeURI needs changed to encodeURIComponent

https://github.com/cosmicjs/cosmic-sdk-js/blob/df94d669b6ae2267fab7478955f0c725ff8c8013/src/clients/bucket/objects/index.ts#L28

**Comment by tonyspiro** [comment]
Thanks for the report. We are looking into this.

**Comment by tonyspiro** [comment]
@facingmonday this is now fixed in `v1.0.10`. Please run `npm i @cosmicjs/sdk@latest`, test it, and let me know if you run into any issue. I'll go ahead and close this issue.
